### PR TITLE
Import previous knative blogs from medium

### PR DESCRIPTION
**Pull request by RichieEscarez**
Clean out formatting and CSS from previous blog posts on medium.com/knative and align with knative.dev look and feel:

+ Manual copy of 04 announcement

+ Add copies of other posts (articles)
   
+ Local copies of graphics

+ Lowercase names for all files

+ Format HTML (ran sublime "reindent")

fixes: #1084

### CODE REVIEW

**Comment by RichieEscarez** [comment]
cleaning up links

File: blog/releases/announcing-knative-v0-4-release.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
...ttps://www.knative.dev/docs/community/README.md#questions-and-issues" data-href="https://www.knative.dev/docs/community/README.md#questions-and-issues" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">Questions and issues</a></li><li name="19ce" id="19ce" class="graf graf--li graf-after--li">Knative on Twitter (<a href="https://twitter.com/KnativeProject" data-href="https://twitter.com/KnativeProject" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">@KnativeProject</a>)</li><li name="5fcb" id="5fcb" class="graf graf--li graf-after--li">Knative on <a href="https://stackoverflow.com/questions/tagged/knative" data-href="https://stackoverflow.com/questions/tagged/knative" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">StackOverflow</a></li><li name="3813" id="3813" class="graf graf--li graf-after--li graf--trailing">Knative <a href="https://slack.knative.dev/" data-href="https://slack.knative.dev/" class="markup--anchor markup--li-anchor" rel="nofollow noopener nofollow noopener" target="_blank">Slack</a></li></ul></div></div></section>
```
Comment (samodell) [inline blog/releases/announcing-knative-v0-4-release.html]: This has some of the same link issues as #1072  -- some links to https://github.com/knative/docs
that should probably not point there.

**Comment by RichieEscarez** [comment]
rebased

**Comment by RichieEscarez** [comment]
going to add other exported blogs re: #1084

**Comment by RichieEscarez** [comment]
Tried to get git to recognize the filename change but it just wants to delete old file and add new.

Otherwise, manually imported the last of the blog files in. PTAL

**Pull request merged**